modules/340-monitoring-yandex-cloud/monitoring/grafana-dashboards/kubernetes-cluster/nat-instance.json

```diff
@@ -0,0 +1,844 @@
+{
```

Comment:
Deploy only if Cluster layout equals to `WithNatInstance`.

Reply:
Do not forget to add an exclusion to the lining rule https://github.com/deckhouse/deckhouse/blob/main/testing/matrix/linter/rules/modules/monitoring.go

Reply:
done